doc/lsp/vim.rst

```diff
@@ -62,7 +90,7 @@ You can try adding ``$`` to the list of keywords to solve the problem:
 
 ::
 
-   autocmd FileType php set iskeyword+=$
+    autocmd FileType php set iskeyword+=$
```

Comment:
unintended indentation?

Reply:
I'm too noob with Github pullrequests, fixed and sorry